eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -78,6 +81,14 @@ public String hash() {
         return result;
     }
 
+    /**
+     * Short version of hash.
+     * @return SHA of commit
+     */
+    public String shortHash() {
+        return this.hash().substring(0, 7);
```

Comment:
@mximp if you introduce this feature here, make sure all places that now do `substring(0, 7)` start using it. Otherwise, we have code duplication: we calculate short Git hash here and somewhere else.

Reply:
fixed